opendbc/safety/tests/test_body.py

```diff
@@ -55,6 +55,15 @@ def test_can_flasher(self):
     self.assertFalse(self._tx(common.make_msg(0, 0x250, dat=b'\xce\xfa\xad\xde\x1e\x0b\xb0')))  # not correct data/len
     self.assertFalse(self._tx(common.make_msg(0, 0x251, dat=b'\xce\xfa\xad\xde\x1e\x0b\xb0\x0a')))  # wrong address
 
+  def test_can_flasher_invalid(self):
+    # Invalid CAN flasher message
+    self.safety.set_controls_allowed(False)
+    self.assertTrue(self._tx(common.make_msg(0, 0x1, 8)))
+    self.assertFalse(self._tx(common.make_msg(0, 0x250, dat=b'\xce\xfa\xad\xde\x1e\x0b\xb0\x1a')))
+
+    self.assertTrue(self._tx(common.make_msg(0, 0x250, dat=b'\xce\xfa\xad\xde\x1e\x0b\xb0\x0a')))
+    # cover test but this could be a valid message(bytes in the CAN firmware)
```

Comment:
I don't understand this comment

Reply:
Just saying is asserting true for the test, and the reason why is true is that this could represent a real byte sequence in a can flasher/firmware payload.